integration-tests/data/allMumbaiTestCases.ts

```diff
@@ -0,0 +1,75 @@
+import { ForgeParams } from '@taquito/local-forging';
```

Comment:
Taken from previous commit, we tend to export new protocol test case in the same file of `allTestsCases.ts`. This is  how it was structured https://github.com/ecadlabs/taquito/blob/master/integration-tests/data/allTestsCases.ts#L1391

Reply:
ahh i see! thanks. i thought it should be there but then second guessed that when i added it to the common cases
will fix